flake.nix

```diff
@@ -74,7 +78,10 @@
               };
               modules = [
                 (self.modules + /top-level.nix)
-                { devenv.warnOnNewVersion = false; }
+                {
+                  packages = [ self.packages.${pkgs.system}.devenv ];
```

Comment:
I think it makes sense that in a Flake context the devenv input Flake defines the version used, not `devenv.yaml`'s `devenv` input. This way developers don't need to globally install devenv either, it remains project-scoped.